.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @holgiosalos @valeryibarra @aperdomob @manuelq12 @JuanDelahozMiranda
```

Comment:
please use these reviewers
@holgiosalos @danielgalvis98 @kliver98 @AlejaGonzalezV @NicolasB2 @manuelq12 @Valeryibarra @veronicatofino

Reply:
Do I have to modify the CODEOWNERS file? or just add them as colaborators on GitHub?

Reply:
Code owners file @Lilainita